bindgen/defines/DefineFinder.cpp

```diff
@@ -109,60 +111,63 @@ void DefineFinder::addNumericConstantDefine(const std::string &macroName,
                                             const clang::Token &token,
                                             bool positive) {
     clang::NumericLiteralParser parser(literal, token.getLocation(), pp);
-    std::string type;
+    Type *type = nullptr;
```

Comment:
This method could be simpler by using `std::string typeName` and still use `"native.CLongLong` etc. and do the `new PrimitiveType(typeName)` allocation at the end.

Reply:
:+1: